daemon/api_systems.go

```diff
@@ -36,8 +36,9 @@ var systemsCmd = &Command{
 }
 
 var systemsActionCmd = &Command{
-	Path: "/v2/systems/{label}",
-	POST: postSystemsAction,
+	Path:     "/v2/systems/{label}",
+	POST:     postSystemsAction,
+	RootOnly: true,
```

Comment:
Should TestUsersOnlyRoot be extended (and possibly renamed) to cover this?

Reply:
Hmm, I'll add something better.